src/api/app/models/concerns/report_bug_url.rb

```diff
@@ -0,0 +1,37 @@
+module ReportBugUrl
+  extend ActiveSupport::Concern
+
+  def report_bug_or_bugzilla_url
+    return report_bug_url if Flipper.enabled?(:foster_collaboration, User.possibly_nobody) && report_bug_url.present?
+
+    return '' if ::Configuration.bugzilla_url.blank?
+
+    bugowners_emails_list = retrieve_bugowners_emails_list
+    return '' if bugowners_emails_list.blank?
+
+    description = if is_a?(Package)
+                    "#{project.name}/#{name}: Bug"
+                  else
+                    "#{name}: Bug"
+                  end
+    bugzilla_url(bugowners_emails_list, description)
+  end
+
+  private
+
+  def retrieve_bugowners_emails_list
+    return (bugowner_emails + project.bugowner_emails).uniq if is_a?(Package)
```

Comment:
If you move  `bugowner_emails` to `Package` and `Project`  then you don't need to distinguish in this concern which also means you can just use it directly in `bugzilla_url` etc.

Reply:
I created another commit with the `bugowner_emails` method refactoring. Please take a look again.